### PR TITLE
LPS-131706 Remove System company usages and tests

### PR DESCRIPTION
**Pull request by adolfopa**

### CODE REVIEW

**Comment by adolfopa** [comment]
ci:test:sf

**Comment by adolfopa** [comment]
ci:test:relevant

**Comment by adolfopa** [comment]
ci:stop

**Comment by adolfopa** [comment]
ci:test:sf

**Comment by adolfopa** [comment]
ci:test:relevant

**Comment by AliciaGarciaGarcia** [comment]
see https://github.com/liferay-lima/liferay-portal/pull/1851

File: portal-impl/src/com/liferay/portal/service/impl/ImageLocalServiceImpl.java [hunk 1]
```diff
@@ -265,6 +330,23 @@ protected void validate(String type) throws PortalException {
 		}
 	}
 
+	private long _getImageCompanyId(long imageId) {
+		Image image = getImage(imageId);
```
Comment (AliciaGarciaGarcia) [inline portal-impl/src/com/liferay/portal/service/impl/ImageLocalServiceImpl.java:334]: In the case of the initial logo, there is no image(on DB) so always will be null

**Pull request closed without merging**